guides/doc-Provisioning_Guide/topics/Bare_Metal.adoc

```diff
@@ -256,6 +256,107 @@ ifeval::["{build}" == "satellite"]
+ifeval::["{build}" == "satellite"]
+. Click the *Parameters* tab, and ensure that a parameter exists that provides an activation key. If not, add an activation key.
+endif::[]
+ifeval::["{build}" == "foreman"]
+. If you use the Katello plugin, click the *Parameters* tab, and ensure that a parameter exists that provides an activation key. If not, add an activation key.
+endif::[]
+. Click *Submit* to save the host details.
++
+For more information about network interfaces, see {BaseURL}/managing_hosts/index#Adding_Network_Interfaces[Adding network interfaces].
+. Start the host and set its boot mode to UEFI HTTP.
+. Select the *Kickstart default PXEGrub2 HTTP Boot*. Until BZ#[BZ#] is resolved, selecting *Kickstart Default PXEGrub2* option fails the provisioning process. Until BZ#[] is resolved, ignore the *error: unrecognized network address* warnings that are displayed during boot. 
+
+This creates the host entry and the relevant provisioning settings. This also includes creating the necessary directories and files for UEFI booting the bare metal host. When you start the physical host and set its boot mode to UEFI HTTP, the host detects the defined DHCP service and starts installing the operating system from its Kickstart tree.
+
+ifeval::["{build}" == "satellite"]
+When the installation completes, the host also registers to {ProjectServer} using the activation key and installs the necessary configuration and management tools from the {project-client-name} repository.
+endif::[]
+
+ifeval::["{build}" == "foreman"]
+If you use the Katello plug-in, when the installation completes, the host also registers to {ProjectServer} using the activation key and installs the necessary configuration and management tools from the {project-client-name} repository.
+endif::[]
+
+.For CLI Users
+
+. On {Project}, enable the `foreman-proxy-http` and `foreman-proxy-httpboot` plug-ins. Disable system checks with the `--disable-system-checks`  and `--skip-checks-i-know-better` options.
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {installer-scenario} \
+--foreman-proxy-httpboot true \
+--foreman-proxy-http true \
+--disable-system-checks \
+--skip-checks-i-know-better
+----
+
+. Create the host with the `hammer host create` command.
++
+[options="nowrap" subs="+quotes"]
+----
+# hammer host create --name "_My_Host_" \
+--organization "_My_Organization_" \
+--location "_My_Location_" \
+--hostgroup "_My_Host_Group_" \
+--mac "_aa:aa:aa:aa:aa:aa_" \
+--build true \
+--enabled true \
+--managed true \
+--pxe-loader "Grub2 UEFI HTTP"
+----
+
+. Ensure the network interface options are set using the `hammer host interface update` command.
++
+[options="nowrap" subs="+quotes"]
+----
+# hammer host interface update --host "_My_Host_" \
+--managed true \
+--primary true \
+--provision true
+----
+
+. Start the host and set its boot mode to UEFI HTTP.
```

Comment:
Start the host, ensure it is set to boot in UEFI mode from network.

Reply:
Good point, will do